package.json

```diff
@@ -1,10 +1,13 @@
 {
   "name": "webmaker.org",
   "version": "1.1.129",
+  "engines": {
```

Comment:
Making it so that webmaker is deployable on heroku seems like something
others can benefit from.  Is there any downside to keeping the `engines`
bit?